### PR TITLE
lightning: refined description about duplicate-resolution

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -128,12 +128,9 @@ addr = "172.16.31.10:8287"
...
 
 # 物理导入模式设置是否检测和解决重复的记录（唯一键冲突）。
-# 目前支持三种解决方法：
-#  - record: 数据写入目标表后，将目标表中重复记录添加到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用下面的 'none' 模式。
+# 目前支持两种解决方法：
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
-#  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
-# duplicate-resolution = 'none'
+#  - remove:如果写入的数据 A 和目标表的数据 B 存在 Primary Key，Unique Key 冲突，则会将 A、B 这两条冲突数据从目标表移除，同时记录到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。用户可根据业务需求选择正确的记录重新手动写入到目标表中。 注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用上面的 'none' 模式。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:133]:
```suggestion
#  - remove：如果写入的数据 A 和目标表的数据 B 存在 Primary Key 或 Unique Key 冲突，则会将 A 和 B 这两条冲突数据从目标表移除，同时记录到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。你可以根据业务需求选择正确的记录重新手动写入到目标表中。注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用上面的 'none' 模式。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -128,11 +128,9 @@ addr = "172.16.31.10:8287"
 # on-duplicate = "replace"
 
 # 物理导入模式设置是否检测和解决重复的记录（唯一键冲突）。
-# 目前支持三种解决方法：
-#  - record: 数据写入目标表后，将目标表中重复记录添加到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用下面的 'none' 模式。
+# 目前支持两种解决方法：
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
-#  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
+#  - remove：如果写入的数据 A 和目标表的数据 B 存在 Primary Key 或 Unique Key 冲突，则会将 A 和 B 这两条冲突数据从目标表移除，同时记录到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。你可以根据业务需求选择正确的记录重新手动写入到目标表中。注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用 'none' 模式。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:133]: 默认值是哪个？

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -128,11 +128,12 @@ addr = "172.16.31.10:8287"
...
-#  - record: 数据写入目标表后，将目标表中重复记录添加到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用下面的 'none' 模式。
+# 目前支持两种解决方法：
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
-#  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
+#  - remove：如果写入的数据 A 和目标表的数据 B 存在 Primary Key 或 Unique Key 冲突，
+#            则会将 A 和 B 这两条冲突数据从目标表移除，同时记录到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
+#            你可以根据业务需求选择正确的记录重新手动写入到目标表中。注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。
+#            如果版本过低，则会启用 'none' 模式。
 # duplicate-resolution = 'none'
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:137]: 默认值是哪个？

Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:137]: 默认值是 “none”

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -128,11 +128,12 @@ addr = "172.16.31.10:8287"
...
-# 目前支持三种解决方法：
-#  - record: 数据写入目标表后，将目标表中重复记录添加到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用下面的 'none' 模式。
+# 目前支持两种解决方法：
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
-#  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
+#  - remove：如果写入的数据 A 和目标表的数据 B 存在 Primary Key 或 Unique Key 冲突，
+#            则会将 A 和 B 这两条冲突数据从目标表移除，同时记录到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
+#            你可以根据业务需求选择正确的记录重新手动写入到目标表中。注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。
+#            如果版本过低，则会启用 'none' 模式。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:136]:
```suggestion
#            如果版本过低，则会启用 'none' 模式。
# 默认值为 'none'。
```

**Review by hfxsd — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -128,11 +128,13 @@ addr = "172.16.31.10:8287"
...
+# 默认值为 'none'。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:137]:
```suggestion
# 默认值为 'none'。 
```

**Review by lance6716 — APPROVED** [review]

**Review by lance6716 — COMMENTED** [review]
/hold

File: tidb-lightning/tidb-lightning-configuration.md [hunk 6]
```diff
@@ -128,11 +128,13 @@ addr = "172.16.31.10:8287"
 # on-duplicate = "replace"
 
 # 物理导入模式设置是否检测和解决重复的记录（唯一键冲突）。
-# 目前支持三种解决方法：
-#  - record: 数据写入目标表后，将目标表中重复记录添加到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用下面的 'none' 模式。
+# 目前支持两种解决方法：
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
-#  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
+#  - remove：如果写入的数据 A 和目标表的数据 B 存在 Primary Key 或 Unique Key 冲突，
```
Comment (lance6716) [inline tidb-lightning/tidb-lightning-configuration.md:133]: A 和 B 可能都是本地导入写入的

```suggestion
#  - remove：如果写入的数据 A 和 B 存在 Primary Key 或 Unique Key 冲突，
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 7]
```diff
@@ -128,11 +128,13 @@ addr = "172.16.31.10:8287"
 # on-duplicate = "replace"
 
 # 物理导入模式设置是否检测和解决重复的记录（唯一键冲突）。
-# 目前支持三种解决方法：
-#  - record: 数据写入目标表后，将目标表中重复记录添加到目标 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。
-#            注意，该方法要求目标 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用下面的 'none' 模式。
+# 目前支持两种解决方法：
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-configuration.md:132]:
```suggestion
#  - none: 不检测重复记录。该模式是两种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Pull request merged**